nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_depth.py

```diff
@@ -112,6 +112,8 @@ def get_search_space(self) -> ElasticDepthSearchSpace:
                 valid_depth_configs.append(list(d_sample))
         self._cached_search_space = valid_depth_configs
         self._is_search_space_obsolete = False
+        if [] not in valid_depth_configs:
```

Comment:
This is needed because the depth search space is missing this configuration. Is it possible to add always the empty configuration to the search space so we can remove this if statement? Thanks!

Reply:
agree, would be better to include it to the search space